client/layout/logged-out.jsx

```diff
@@ -151,6 +154,7 @@ export default connect( ( state ) => {
 	const noMasterbarForSection = 'signup' === section.name || 'jetpack-connect' === section.name;
 	const isJetpackWooCommerceFlow =
 		'woocommerce-onboarding' === get( getCurrentQueryArguments( state ), 'from' );
+	const isJetpackWooDnaFlow = wooDnaConfig[ get( getCurrentQueryArguments( state ), 'from' ) ];
```

Comment:
I'd expect this variable to be a boolean, not an object or undefined. Should we wrap this in `Boolean()`?

Reply:
Sure. It's used as a bolean, so it doesn't hurt to make it explicit. Done in https://github.com/Automattic/wp-calypso/pull/41798/commits/a9861190a0922bb53497ea7e0fde3565fe7efd2d